src/controllers/ControllerComplaint.ts

```diff
@@ -107,6 +108,30 @@ export default class ControllerComplaint {
 		}
 	}
 
+	async removeVote(req: Request, res: Response): Promise<Response> {
+		try {
+			const vote = await this.complaintRepository.getComplaintById(
+				Number(req.query.userId),
```

Comment:
Um pedido que gostaria de fazer.
Por uma questão de legibilidade na leitura do código, não passem como parâmetro direto o `req.query.ALGUMACOISA`.
Atribua os parâmetros da requisição a uma constante antes de você começar a utilizar o valor.

Por exemplo:
Coloquem algo como:
`const userId = Number(req.query.userId)`

Isso facilita na leitura de quem for revisar esse código posteriormente